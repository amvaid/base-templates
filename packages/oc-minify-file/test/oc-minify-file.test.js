// 'use strict';

// const expect = require('chai').expect;

// describe('cli : domain : package-static-files : minify-file', () => {
//   const minifyFile = require('../../src/cli/domain/package-static-files/minify-file');

//   describe('when minifying .js file', () => {

//     describe('when file contains es6', () => {

//       const content = 'const hi = (name) => `hello ${name}`;';

//       xit('should minify it', () => {
//         const minified = minifyFile('.js', content);
//         expect(minified).to.equal('var hi=function(n){return"hello "+n};');
//       });
//     });

//     describe('when file contains not valid js', () => {
//       const content = 'const a=notvalid(';
//       const execute = function(){
//         minifyFile('.js', content);
//       };

//       xit('should throw an exception', () => {
//         expect(execute).to.throw('Unexpected token');
//       });
//     });
//   });
// });
