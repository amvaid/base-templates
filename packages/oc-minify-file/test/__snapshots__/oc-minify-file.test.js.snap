// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Minify es6 files 1`] = `"var hi=function(n){return\\"hello \\"+n};"`;

exports[`Minify not valid css 1`] = `"Css is not valid"`;

exports[`Minify not valid js 1`] = `"unknown: Unexpected token (1:17)"`;

exports[`Minify valid css file 1`] = `"body{background-color:#fff}"`;
